src/lib/net/internal/index.ts

```diff
@@ -0,0 +1,107 @@
+import { Players, RunService } from "@rbxts/services";
+
+import { EventListener } from "../../events";
+import { FunctionUtil } from "../../util/functions";
+import { Identifiers } from "../util/identifiers";
+import { ClientListener, Packet, ServerListener } from "../util/types";
+import { InternalNetworkingEvents } from "./types";
+
+export namespace Internals {
+	let remote: RemoteEvent | undefined;
+
+	const listeners: { [x: string]: Array<ClientListener | ServerListener> } = {};
```

Comment:
Can we use a Map here?

Reply:
Sure, but I don't see the why to. Simple change either way.